process_report/processors/lenovo_processor.py

```diff
@@ -9,8 +9,18 @@
 class LenovoProcessor(processor.Processor):
     SU_CHARGE_MULTIPLIER = 1
```

Comment:
This should be removed now, right? Since this used to be the charge that we got.